spec/factories/users.rb

```diff
@@ -0,0 +1,8 @@
+FactoryBot.define do
+  factory :user do
+    username { Faker::Internet.username(specifier: 4..9) }
```

Comment:
As a funny note, sometimes Faker names can duplicate and cause some unique name error

Reply:
yeah, that's funny xd, I think they already know about that issue, they address it in their [docs](https://github.com/faker-ruby/faker#ensuring-unique-values)